frontend/src/app/utils/disease.ts

```diff
@@ -5,8 +5,12 @@ import { MULTIPLEX_DISEASES } from "../testResults/constants";
 export const useSupportedDiseaseList = () => {
   let allDiseases = Object.values(MULTIPLEX_DISEASES);
   const hivEnabled = Boolean(useFeature("hivEnabled"));
+  const syphilisEnabled = Boolean(useFeature("syphilisEnabled"));
   if (!hivEnabled) {
     allDiseases = allDiseases.filter((d) => d !== MULTIPLEX_DISEASES.HIV);
   }
+  if (!syphilisEnabled) {
+    allDiseases = allDiseases.filter((d) => d !== MULTIPLEX_DISEASES.SYPHILIS);
+  }
```

Comment:
I thought about trying to do something fancy here instead of the two conditional blocks but it seemed like overkill given that we only have two cases right now. If anyone has a sleeker solution that is also straight forward though please let me know

Reply:
heads up you / mike will probably have a merge conflict dependent on when [his work gets in](https://github.com/CDCgov/prime-simplereport/pull/7539/files#diff-8d5caf8dda8301fd580750920553cdeb93629bffef393fbe1599849608e7c004R11)

Reply:
Looks like it's almost exactly the same besides which line syphilisEnabled is declared on so should be real quick to resolve if it doesn't auto resolve. As for the conditional blocks I think it's good as is! When we remove a feature flag it's simple to remove that block and likewise for adding a new feature flag for a future disease like Hep C